playbooks/99-logs.yml

```
@@ -34,6 +34,17 @@
             cifmw_path: "{{ ansible_user_dir }}/.crc/bin:{{ ansible_user_dir }}/.crc/bin/oc:{{ ansible_user_dir }}/bin:{{ ansible_env.PATH }}"
             cacheable: true
 
+    - name: Set destination folder for the logs
+      ansible.builtin.set_fact:
+        logfiles_dest_dir: >-
+          {{
+            (
+              cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data'),
+              '/logs/',
```

```suggestion
              'logs/',
```

else the built path is `/logs/2024-04-17_13-55`